app/src/main/java/org/septa/android/app/database/SEPTADatabase.java

```diff
@@ -14,37 +14,73 @@
     /**
      * Current packaged DB version, update number when packaged DB changes
      */
-    private static final int DATABASE_VERSION = 282;
-    private static final String DATABASE_FILE_NAME = "SEPTA.sqlite";
+    private static final String LATEST_DATABASE_API_URL = "https://s3.amazonaws.com/mobiledb.septa.org/latest/latestDb.json";
```

Comment:
Externalize this String into Values.

Reply:
I have been putting strings that are displayed to the user into string resources. (Supposing the app had multiple languages, localization would be useful here.) Strings only used internally or for the code are better kept as string constants.